ðŸ“Ÿ Â Training on MPS
Traceback (most recent call last):
  File "/Users/arthur/Library/Mobile Documents/com~apple~CloudDocs/Cours/ECN6338/Devoir aÌ€ rendre/Remise/train.py", line 335, in <module>
    train(cfg)
    ~~~~~^^^^^
  File "/Users/arthur/Library/Mobile Documents/com~apple~CloudDocs/Cours/ECN6338/Devoir aÌ€ rendre/Remise/train.py", line 227, in train
    with Path(cfg.data_path).open("rb") as fh:
         ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/arthur/miniconda3/envs/ECN6338/lib/python3.13/pathlib/_local.py", line 537, in open
    return io.open(self, mode, buffering, encoding, errors, newline)
           ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data\\data.pickle'
